### PR TITLE
[12.0][ADD] currency_rate_update_oxr: OpenExchangeRates.org provider

### PR DESCRIPTION
**Pull request by alexey-pelykh**
This module provides adds OpenExchangeRates.org currency exchange rates provider.

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Please check Travis

**Comment by alexey-pelykh** [comment]
@pedrobaeza fixed

**Review by pedrobaeza — APPROVED** [review]

File: currency_rate_update_oxr/readme/DESCRIPTION.rst [hunk 1]
```diff
@@ -0,0 +1,2 @@
+This module provides adds `OpenExchangeRates.org <https://openexchangerates.org/>`_
```
Comment (jcdrubay) [inline currency_rate_update_oxr/readme/DESCRIPTION.rst]: Remove `provides`

File: currency_rate_update_oxr/README.rst [hunk 2]
```diff
@@ -0,0 +1,82 @@
+============================
+Currency Rate Update: XE.com
```
Comment (jcdrubay) [inline currency_rate_update_oxr/README.rst]: Should be OpenExchangeRates.org

File: currency_rate_update_oxr/README.rst [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Configuration
+=============
+
+To configure XE.com currency rates provider credentials:
```
Comment (jcdrubay) [inline currency_rate_update_oxr/README.rst]: OpenExchangeRates.org

File: currency_rate_update_oxr/README.rst [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+
+# Go to *Invoicing > Configuration > Settings*
+# Fill application credentials in *Currencies > XE.com Provider* section
```
Comment (jcdrubay) [inline currency_rate_update_oxr/README.rst]: OpenExchangeRates.org

**Review by jcdrubay — CHANGES_REQUESTED** [review]
LGTM, only small updates to do for repleacement of XE by OpenExchangeRates.org

File: currency_rate_update_oxr/readme/DESCRIPTION.rst (hunk 1 above)
Comment (alexey-pelykh) [reply currency_rate_update_oxr/readme/DESCRIPTION.rst]: Fixed, thanks!

File: currency_rate_update_oxr/README.rst (hunk 2 above)
Comment (alexey-pelykh) [reply currency_rate_update_oxr/README.rst]: Regenerated

File: currency_rate_update_oxr/README.rst (hunk 4 above)
Comment (alexey-pelykh) [reply currency_rate_update_oxr/README.rst]: Regeneratd

**Review by Syentystech1 — APPROVED** [review]

**Review by jcdrubay — APPROVED** [review]

**Pull request merged**